app/tps_counter_test.go

```diff
@@ -46,8 +46,8 @@ func TestTPSCounter(t *testing.T) {
 	// We expect that the TPS reported will be:
 	// 100 / 5ms => 100 / 0.005s = 20,000 TPS
 	lines := strings.Split(buf.String(), "\n")
-	require.Equal(t, repeat+1, len(lines), "Expected exactly n repeats")
-	wantReg := regexp.MustCompile("Transactions per second tps \\d+\\.\\d+")
+	require.True(t, len(lines) > 1, "Expected at least 1 line")
```

Comment:
"At least 1 line" means `len(lines) >= 1` not `len(lines) > 1`?

Reply:
You are right, it should be `>=` in comparison or `"Expected more than 1 line"` as error string